docs/dev/connectors/kafka.md

```diff
@@ -713,14 +713,14 @@ the consumers until `transaction1` is committed or aborted. This has two implica
 **Note**:  `Semantic.EXACTLY_ONCE` mode uses a fixed size pool of KafkaProducers
 per each `FlinkKafkaProducer011` instance. One of each of those producers is used per one
 checkpoint. If the number of concurrent checkpoints exceeds the pool size, `FlinkKafkaProducer011`
-will throw an exception and will fail the whole application. Please configure max pool size and max
-number of concurrent checkpoints accordingly.
+will throw an exception and will fail the whole application. Please configure max pool size(`FlinkKafkaProducer011.kafkaProducersPoolSize`) 
+and max number of concurrent checkpoints(`CheckpointConfig.maxConcurrentCheckpoints`) accordingly.
```

Comment:
the reference to `CheckpointConfig` doesn't belong here. Instead there should be a paragraph for this feature in the relevant section of the documentation (Checkpoints/Tuning checkpointing) that we refer to here. There's a small note [here](https://ci.apache.org/projects/flink/flink-docs-release-1.7/ops/state/large_state_tuning.html#tuning-checkpointing) that we could extend instead.